client/cypress/integration/shoppinglist.spec.ts

```diff
@@ -89,10 +89,37 @@ describe('Delete from Shopping List', () => {
     cy.wait(1000);
   });
 
-  it('should click the delete button on an item and confirm delete', () => {
-    page.clickDeleteButton(0);
-    page.clickDialogDeleteButton();
-    page.getStoreItems(0).should('have.length', 3);
+  describe('Deleting from the left-most store', () => {
+    const storePos = 0;
+    let originalNumItems: number;
+
+    beforeEach(() => {
+      // Is this really a reasonable way to do this?
```

Comment:
I'm not thrilled about this way of capturing `originalNumItems` and would be happy to take suggestions for alternatives.